src/v2/Apps/Artist/Routes/WorksForSale/Utils/getWorksForSaleRouteVariables.ts

```diff
@@ -2,16 +2,27 @@ import { paramsToCamelCase } from "v2/Components/ArtworkFilter/Utils/urlBuilder"
 import { initialArtworkFilterState } from "v2/Components/ArtworkFilter/ArtworkFilterContext"
 import { allowedFilters } from "v2/Components/ArtworkFilter/Utils/allowedFilters"
 
-export function getWorksForSaleRouteVariables({ artistID }, props) {
+export function getWorksForSaleRouteVariables(
+  { artistID },
+  { location, context }
+) {
+  const isMerchandisingEnabled = !!context.featureFlags?.["merchandising-test"]
```

Comment:
If we try to use `getENV("FEATURE_FLAGS")`, we will get `undefined` during server rendering